audio-worklet/src/noisegate-demo.js

```diff
@@ -187,6 +206,19 @@ class NoisegateDemo {
     }
   }
 
+  workletSelected() {
+    // Switching occurs 10ms into the future for thread synchronization.
```

Comment:
If you want to take account of different latency between SPN and AWN, the value should not be `10ms`. I suggest to remove 0.01s here and simply use the current time for both. I don't expect A-Bing to be completely glitchless.